.github/workflows/test.yml

```diff
@@ -85,17 +107,25 @@ jobs:
           # django versions that support py3.10
           - python-version: '3.10'
             django-version: 3.2
+          - python-version: '3.10'
+            django-version: 4.0
+
+          # django versions that support py3.11
+          - python-version: '3.11'
+            django-version: 4.1
+          - python-version: '3.11'
+            django-version: 4.2
```

Comment:
these are still supported, move to `functional`

Reply:
Same here.